deployment/changeset/v1_6/chain.go

```diff
@@ -0,0 +1,39 @@
+package v1_6
+
+type ChainDefinition struct {
+	// ConnectionConfig holds configuration for connection.
+	ConnectionConfig
+	// Selector is the chain selector of this chain.
+	Selector uint64
+	// FeeQuoterDestChainConfig is the configuration to be applied on Source chain when this chain is a destination.
+	FeeQuoterDestChainConfig FeeQuoterDestChainConfig
```

Comment:
Can we ensure that there will be a fee quoter with all the remote chain config on every chain? Or should this be more generic and just be `DestChainConfig` or `RemoteChainConfig`?